### ISSUE TITLE
Saving certificates causes app to hang

### ISSUE THREAD

**Issue by ddag** [openconnect/openconnect-gui#248]
I'm using a user certificate. When I specify both the cert + key and hit save, the app hangs. It stores the CA cert fine, but always hangs on the user cert. Any ideas ?

**Comment by ddag** [comment]
Oh, this is Windows 10 latest updates.

**Comment by horar** [comment]
Yes, I think - this is duplicate to #128; will be fixed with next release...
